db/seeds.rb

```diff
@@ -216,3 +216,7 @@
             dsi_data: { "school_urns" => %w[137138 144523], "trust_uids" => %w[16644], "la_codes" => %w[852] },
             family_name: "Carter",
             given_name: "Valentine")
+
+Jobseeker.create(email: "jobseeker@example.com",
+                 password: "password",
```

Comment:
Does this work? I'd be doubtful, surely the hashed password is what needs to be saved in the database?

Reply:
It does work 🤔

Reply:
Devise magic! It hashes the password on save 🧙